templates/item/partials/ability.hbs

```diff
@@ -32,6 +32,7 @@
 </fieldset>
 {{formGroup systemFields.powerRoll.fields.enabled value=system.powerRoll.enabled}}
 {{#if system.powerRoll.enabled}}
+{{formGroup systemFields.powerRoll.fields.characteristics value=system.powerRoll.characteristics}}
```

Comment:
Add `context.characteristics = Object.entries(ds.CONFIG.characteristics).map(([value], {label}) => ({value, label}));` to `getSheetContext` and then add `options=characteristics` in this